/oak:
  ros__parameters:
    camera:
      i_enable_imu: true
      i_enable_ir: true
      i_floodlight_brightness: 0
      i_ip: ''
      i_laser_dot_brightness: 0
      i_max_q_size: 4
      i_mx_id: ''
      i_nn_type: rgb
      i_pipeline_type: RGBD
      i_usb_speed: SUPER_PLUS
    color:
      i_board_socket_id: 0
      i_enable_preview: true
      i_fps: 30.0
      i_height: 1080
      i_interleaved: false
      i_keep_preview_aspect_ratio: true
      i_max_q_size: 4
      i_preview_size: 256
      i_publish_topic: true
      i_resolution: '1080'
      i_set_isp: false
      i_width: 1920
      r_exposure: 20000
      r_focus: 1
      r_iso: 800
      r_set_man_exposure: false
      r_set_man_focus: false
      r_set_man_whitebalance: false
      r_whitebalance: 3300
    left:
      i_board_socket_id: 1
      i_fps: 30.0
      i_max_q_size: 4
      i_publish_topic: true
      i_resolution: '720'
      r_exposure: 1000
      r_iso: 800
      r_set_man_exposure: false
    nn:
      i_label_map:
      - background
      - aeroplane
      - bicycle
      - bird
      - boat
      - bottle
      - bus
      - car
      - cat
      - chair
      - cow
      - diningtable
      - dog
      - horse
      - motorbike
      - person
      - pottedplant
      - sheep
      - sofa
      - train
      - tvmonitor
      i_model_path: /workspace/install/depthai_ros_driver/share/depthai_ros_driver/models/deeplab_v3_plus_mnv2_decoder_256_openvino_2021.4.blob
      i_nn_config_path: /workspace/install/depthai_ros_driver/share/depthai_ros_driver/config/nn/segmentation.json
      i_num_inference_threads: 2
      i_num_pool_frames: 4
    right:
      i_board_socket_id: 2
      i_fps: 30.0
      i_max_q_size: 4
      i_publish_topic: true
      i_resolution: '720'
      r_exposure: 1000
      r_iso: 800
      r_set_man_exposure: false
    stereo:
      i_align_depth: true
      i_board_socket_id: 0
      i_decimation_factor: 1
      i_depth_filter_size: 7
      i_depth_preset: HIGH_DENSITY
      i_enable_spatial_filter: true
      i_enable_speckle_filter: false
      i_enable_temporal_filter: true
      i_extender_disp: false
      i_height: 1080
      i_hole_filling_radius: 2
      i_lr_check: true
      i_lrc_threshold: 5
      i_max_q_size: 4
      i_rectify_edge_fill_color: -1
      i_spatal_filter_iterations: 1
      i_speckle_range: 50
      i_stereo_conf_threshold: 255
      i_threshold_filter_max_range: 15000
      i_threshold_filter_min_range: 400
      i_width: 1920
    use_sim_time: false
